tmd/Tree/methods.py

```diff
@@ -184,15 +195,30 @@ def get_point_path_distances(self):
     return path_lengths
 
 
+def get_point_trunk_length(self):
+    '''Tree method to get the trunk (first section length).
+    '''
+    ways, end = self.get_sections_only_points()
+    first_section_id = np.where(ways == 0)
+    first_section_start = ways[first_section_id]
+    first_section_end = end[first_section_id]
+    seg_nums = range(first_section_start[0] + 1, first_section_end[0] + 1)
+
+    seg_lengths = get_segment_lengths(self, seg_nums)
+    trunk_length = np.sum(seg_lengths)
+
+    return np.repeat(trunk_length, size(self))
```

Comment:
I am not sure here: should we return only one number?
I returned an array with the same value for all points to be consistent with other `get_point_*` functions.

Reply:
No we should return one number. It doesn't make sense to have a "point" function for this. Point functions are only useful for the TMD computation. The trunk measurement should rather be a single value so the TMD doesn't make sense for this measurement.

Reply:
Ok, I will change that.